setup.py

```diff
@@ -154,6 +154,14 @@
     package_dir={"": "src"},
     packages=find_packages(where="src"),
     install_requires=["numpy", "pandas", "scipy"],
-    ext_modules=cythonize(ext_modules, annotate=False),
+    ext_modules=cythonize(
+        ext_modules,
+        annotate=False,
+        compiler_directives={
+            "boundscheck": False,
+            "wraparound": False,
+            "cdivision": True,
```

Comment:
I think it would be helpful to add some way to set boundscheck and wraparound to True when debugging. Suggestion: check for the environment variable "TABMAT_DEBUG" when doing pip install. If set, then turn on the boundscheck and the wraparound.

Reply:
No, using O0 will modify (undefined) behavior and thus you won't be able to debug a lot of problems. Thus, don't set it.